extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -895,6 +900,43 @@ export class PublishDatabaseDialog {
 
 		return true;
 	}
+
+	//#region
+	/*
+	 * Creates Display options container with hyperlink options
+	 */
+	private createOptionsButton(view: azdataType.ModelView): azdataType.FlexContainer {
+		this.optionsButton = view.modelBuilder.button().withProps({
+			label: constants.configureOptions,
+			secondary: true
+		}).component();
+
+		const optionsRow = view.modelBuilder.flexContainer().withItems([this.optionsButton], { CSSStyles: { flex: '0 0 auto', 'margin': '8px 0 0 295px' } }).withLayout({ flexFlow: 'row', alignItems: 'center' }).component();
+
+		this.optionsButton.onDidClick(async () => {
+			TelemetryReporter.sendActionEvent(TelemetryViews.SqlProjectPublishDialog, 'PublishOptionsClicked');
+			// create fresh every time
```

Comment:
wouldn't we want to remember the options in case someone chose some options, closed the dialog, then decided they wanted to change another option and open the dialog again?

Reply:
(AutoSave)Seems Its a nice to have feature, on other hand, if they don't want to save the selection, they need to click the 'Reset' button. We already provide an 'ok' button to manually save the selection. either way user has to go with same number of button clicks. would like to discuss on this one with you guys :)

Reply:
Oh got it! let me update the comment more clearly. Thanks :)